tests/exec.rs

```diff
@@ -7,6 +8,12 @@ fn main() {
     let mut birdcage = Birdcage::new().unwrap();
     birdcage.add_exception(Exception::ExecuteAndRead("/usr/bin/true".into())).unwrap();
     birdcage.add_exception(Exception::ExecuteAndRead("/usr/lib".into())).unwrap();
+    if PathBuf::from("/lib64").exists() {
+        birdcage.add_exception(Exception::ExecuteAndRead("/lib64".into())).unwrap();
+    }
+    if PathBuf::from("/lib64").exists() {
+        birdcage.add_exception(Exception::ExecuteAndRead("/lib".into())).unwrap();
+    }
```

Comment:
This is super ugly, but once landlock is removed we don't actually need to fail on missing paths anymore.

So I plan to remove this and add a test that confirms that all the exceptions work with invalid paths on all platforms, so I think it's fine as a temporarily permanent workaround.